src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -223,6 +223,34 @@ describe('ThreadView', () => {
       expect(JSON.parse(axiosMock.history.patch[axiosMock.history.patch.length - 1].data)).toMatchObject(data);
     }
 
+    it('should not show post footer', async () => {
+      Factory.resetAll();
```

Comment:
don't need to pass Factory.resetAll(); because its already present in the beforeEach. It will automatically call before executing each test case, so you can remove this line

Reply:
Factory needs to be reset for these two test cases otherwise previous fetched API result is used for test cases, not the ones I have fetched explicitly for these test cases